api/v1alpha1/nodepool_types.go

```diff
@@ -651,6 +651,13 @@ type KubevirtDiskImage struct {
 	ContainerDiskImage *string `json:"containerDiskImage,omitempty"`
 }
 
+type MultiQueueSetting string
+
+const (
+	MultiQueueEnable  MultiQueueSetting = "enable"
+	MultiQueueDisable MultiQueueSetting = "disable"
```

Comment:
For consistency, we should use capital letters to start these values. `Enable` `Disable`